models/ed/R/met2model.ED2.R

```diff
@@ -267,7 +267,7 @@ for(year in start_year:end_year) {
 
   ## write DRIVER file
   sites <- 1
-  metgrid <- c(1,1,1,1,lon,lat)
+  metgrid <- c(1,1,1,1,floor(lon),floor(lat))
```

Comment:
Ann, is there a reason for the change in how the ED met header file is written? @robkooper @bcow what did we determine was the correct setting here?